packages/backend/src/tools/JobQueue.ts

```diff
@@ -18,7 +18,9 @@ export class JobQueue {
   private inProgress: JobInQueue[] = []
   private lastUpdatedAt = new Date().toISOString()
 
-  constructor(private options: JobQueueOptions, private logger: Logger) {}
+  constructor(private options: JobQueueOptions, private logger: Logger) {
+    this.logger = logger.for(this)
```

Comment:
Not having this line here was on purpose. JobQueue is not an Application level sevice but a utility. Maybe logger can have a `nested` method, so that it formats as: `ServiceName.JobQueue` instead

Reply:
Hmm... shouldn't `for` always work like this?

If I have a `logger.for(syncScheduler).for(jobQueue)` formatting `ServiceName.JobQueue` seems like the expected result.